mods/ra/maps/exodus/exodus.lua

```diff
@@ -0,0 +1,394 @@
+Difficulty = Map.LobbyOption("difficulty")
+
+UnitsEvacuatedThreshold =
+{
+	hard = 200,
+	normal = 100,
+	easy = 50
+}
+
+AttackAtFrame =
+{
+	hard = 500,
+	normal = 500,
+	easy = 600
+}
+
+MinAttackAtFrame =
+{
+	hard = 200,
+	normal = 100,
+	easy = 50
+}
+
+MaxSovietYaks =
+{
+	hard = 4,
+	normal = 2,
+	easy = 0
+}
+
+SovietParadrops =
+{
+	hard = 40,
+	normal = 20,
+	easy = 0
+}
+
+SovietParadropTicks =
+{
+	hard = DateTime.Minutes(17),
+	normal = DateTime.Minutes(20),
+	easy = DateTime.Minutes(20)
+}
+
+SovietUnits2Ticks =
+{
+	hard = DateTime.Minutes(12),
+	normal = DateTime.Minutes(15),
+	easy = DateTime.Minutes(15)
+}
+
+SovietEntryPoints =
+{
+	SovietEntryPoint1, SovietEntryPoint2, SovietEntryPoint3, SovietEntryPoint4, SovietEntryPoint5, SovietEntryPoint6
+}
+
+SovietRallyPoints =
+{
+	SovietRallyPoint1, SovietRallyPoint2, SovietRallyPoint3, SovietRallyPoint4, SovietRallyPoint5, SovietRallyPoint6
+}
+
+SovietAirfields =
+{
+	SovietAirfield1, SovietAirfield2, SovietAirfield3, SovietAirfield4,
+	SovietAirfield5, SovietAirfield6, SovietAirfield7, SovietAirfield8
+}
+
+MountainEntry =
+{
+	MountainEntry1.Location, MountainEntry2.Location, MountainEntry3.Location, MountainEntry4.Location,
+	MountainEntry5.Location, MountainEntry6.Location, MountainEntry7.Location, MountainEntry8.Location
+}
+
+BridgeEntry =
+{
+	BridgeEntry1.Location, BridgeEntry2.Location, BridgeEntry3.Location, BridgeEntry4.Location
+}
+
+MobileConstructionVehicle = { "mcv" }
+Yak = { "yak" }
+
+ReinforcementsTicks1 = DateTime.Minutes(5)
+Reinforcements1 =
+{
+	"mgg", "2tnk", "2tnk", "2tnk", "2tnk", "1tnk", "1tnk",
+	"jeep", "jeep", "e1", "e1", "e1", "e1", "e3", "e3"
+}
+
+ReinforcementsTicks2 = DateTime.Minutes(10)
+Reinforcements2 =
+{
+	"mgg", "2tnk", "2tnk", "2tnk", "2tnk", "truk", "truk", "truk",
+	"truk",	"truk", "truk", "1tnk", "1tnk", "jeep", "jeep"
+}
+
+SovietUnits1 =
+{
+	"3tnk", "3tnk", "3tnk", "3tnk", "3tnk", "3tnk", "v2rl", "v2rl", "ftrk",
+	"apc", "e1", "e1", "e2", "e3", "e3", "e4"
+}
+
+SovietUnits2 =
+{
+	"4tnk", "4tnk", "4tnk", "4tnk", "3tnk", "3tnk", "3tnk", "3tnk", "v2rl",
+	"v2rl", "ftrk", "apc", "e1", "e1", "e2", "e3", "e3", "e4"
+}
+
+CurrentReinforcement1 = 0
+CurrentReinforcement2 = 0
+SpawnAlliedUnit = function(units)
+	Reinforcements.Reinforce(allies1, units, { Allies1EntryPoint.Location, Allies1MovePoint.Location })
+
+	if allies2 then
+		Reinforcements.Reinforce(allies2, units, { Allies2EntryPoint.Location, Allies2MovePoint.Location })
+	end
+
+	Utils.Do(humans, function(player)
+		if player and player.IsLocalPlayer then
+			Trigger.AfterDelay(DateTime.Seconds(3), function()
+				Media.PlaySpeechNotification(allies1, "AlliedReinforcementsArrived")
```

Comment:
Changed it to "Allied reinforcements have arrived from the North." to avoid confusion.